arachni/Makefile

```diff
@@ -0,0 +1,4 @@
+NAME := arachni
+VERSION: = 1.0.0
```

Comment:
1. typo (`: =` vs ` :=`)
2. Not sure if hardcoding `VERSION` in Makefile makes sense? Dockerhub will normally build and tag `latest` (per [howto](https://github.com/unit9/docklabs/blob/master/howto.md)); also we don't really have a policy for versioning images.